docs/docs/network/02-coordinator.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_position: 2
+description: Ignite Network commands for coordinator.
+---
+
+# Coordinator Guide
+
+Coordinators organize and launch new chains on Ignite Chain.
+
+---
+
+## Publish a chain
+
+The first step in the process of a chain launch is for the coordinator to publish the intention of launching a chain.
+The `publish` command publishes the intention of launching a chain on Ignite from a project git repository.
+
+```shell
+ignite n chain publish https://github.com/ignite/example
+```
+
+**Output**
+
+```
+✔ Source code fetched
+✔ Blockchain set up
+✔ Chain's binary built
+✔ Blockchain initialized
+✔ Genesis initialized
+✔ Network published
+⋆ Launch ID: 3
+```
+
+`LaunchID` identifies the published blockchain on Ignite blockchain.
+
+### Specify a initial genesis
+
+During coordination, new genesis accounts and genesis validators are added into the chain genesis.
+The initial genesis where these accounts are added is by default the default genesis generated by the chain binary.
+
+The coordinator can specify a custom initial genesis for the chain launch with the `--genesis` flag. This custom initial genesis can contain additional default genesis accounts and custom params for the chain modules.
+
+A URL must be provided for the `--genesis` flag. This can either directly point to a JSON genesis file or a tarball containing a genesis file.
+
+```shell
```

Comment:
Some code blocks have `shell` annotation, while some have none.  Let's be consistent.

Reply:
I think we can remove it everywhere